### ISSUE TITLE
Has stopped working for Haaretz.co.il

### ISSUE THREAD

**Issue by ZivGla** [iamadamdev/bypass-paywalls-firefox#336]
Does not bypass Haaretz.co.il paywall in the last few days

**Comment by Enterisr** [comment]
true, especially for articles that requires paid account (and do not let you read as a free user), that fetching the content after the sign in. 
ie:
https://www.haaretz.co.il/magazine/.premium.highlight-MAGAZINE-1.7951256

**Comment by magnolia1234** [comment]
Fixed (only) premium articles in PR #393
Above article gets redirected to premium though.
